static-instrumentation/agent-instrumenter/build.gradle

```diff
@@ -0,0 +1,12 @@
+plugins {
+    id("otel.java-conventions")
+}
+
+ext.libraryName = 'OpenTelemetry Java Contrib Static Instrumentation Agent'
```

Comment:
We don't need `ext.libraryName` here, especially since we have `description`

Reply:
🆗